Spec/RestClient.swift

```diff
@@ -832,7 +832,22 @@ class RestClient: QuickSpec {
                 let object = try! NSJSONSerialization.JSONObjectWithData(transport.rawDataSent.first!, options: NSJSONReadingOptions(rawValue: 0))
                 expect(NSJSONSerialization.isValidJSONObject(object)).to(beTrue())
             }
-
+            
+            // RSC7b
+            it("X-Ably-Lib: [lib][.optional variant]?-[version] should be included in all REST requests") {
+                let client = ARTRest(options: AblyTests.commonAppSetup())
+                client.httpExecutor = testHTTPExecutor
+                let channel = client.channels.get("test")
+                waitUntil(timeout: testTimeout) { done in
+                    channel.publish(nil, data: "message") { error in
+                        expect(error).to(beNil())
+                        let headerLibVersion = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Lib"]
+                        let ablyBundleLibVersion = ARTDefault.libraryVersion()
+                        expect(headerLibVersion).to(equal(ablyBundleLibVersion))
```

Comment:
Whilst I understand that you don't necessarily want the the test to change for every time a version changes, it does feel like this test needs to be a bit more verbose i.e. can it not at least check that the header matches `ios-0.8.*` for example?  It just means the test confirms what we expect as opposed to simply referencing another part of the internals of the lib.